smart-contracts/tasks/utils.ts

```diff
@@ -3,6 +3,8 @@ import SafeApiKit from '@safe-global/api-kit'
 import Safe from '@safe-global/protocol-kit'
 import { Confirm, Select } from 'enquirer'
 
+const MAINNET_SAFE_ADDRESS = '0x1f06b7dd281Ca4D19d3E0f74281dAfDeC3D43963'
```

Comment:
Ha! I guess you added this because `getSafesByOwner` does not return safe for which someone is a submitter?